docs/ug/configFiles.md

```diff
@@ -29,17 +29,17 @@ Given below are the details of the various config files used by RepoSense.
 **`repo-config.csv` file contains repo-level config data.** Each row represents a repository's configuration ([example](repo-config.csv)).
 
 
-Column Name | Explanation
------------ | -----------
-Repository's Location {{ mandatory }}| The `GitHub URL` or `Disk Path` to the git repository e.g., `https://github.com/foo/bar.git` or `C:\Users\user\Desktop\GitHub\foo\bar`
-Branch | The branch to analyze in the target repository e.g., `master`. Default: the default branch of the repo
-File formats<sup>*+</sup> | The file extensions to analyze. Binary file formats, such as `png` and `jpg`, will be automatically labelled as the file type `binary` in the generated report. Default: all file formats
-Find Previous Authors| Enter **`yes`** to utilize Git blame's ignore revisions functionality, RepoSense will attempt to blame the line changes caused by commits in the ignore commit list to the previous authors who altered those lines (if available).
-Ignore Glob List<sup>*+</sup> | The list of file path globs to ignore during analysis for each author e.g., `test/**;temp/**`. Refer to the [_glob format_](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob) for the path glob syntax.
-Ignore standalone config | To ignore the standalone config file (if any) in target repository, enter **`yes`**. If the cell is empty, the standalone config file in the repo (if any) will take precedence over configurations provided in the csv files.
-Ignore Commits List<sup>*+</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash. Additionally, a range of commits can be specified using the `..` notation e.g. `abc123..def456` (both inclusive).
-Ignore Authors List<sup>*+</sup> | The list of authors to ignore during analysis. Authors should be specified by their [Git Author Name](#a-note-about-git-author-name).
-Shallow Cloning | Enter **`yes`** to clone the repository using Git's shallow cloning functionality. This option can significantly reduce the time taken to clone large repositories. However, the option should ideally be disabled for smaller repositories where the `.git` file is smaller than 500 MB, as it would create overhead.
+| Column Name                           | Explanation                                                                                                                                                                                                                                                                                                            |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Is there a difference if the padding is left out? I think it might be quite a lot of extra work to maintain the alignment if these tables were ever updated.

Reply:
This padding separates the headers from the entries. I don't think it should be left out.

Reply:
My bad I didn't make myself clear. I believe that line itself is necessary for markdown tables but I was referring to the padding that results in the bars aligning. 

My Intellij's markdown addon does not actually show any warnings on the current master branch with single `|` separator per line. Perhaps someone else should comment on this.

Reply:
Ah I see. I thought I had set it to automatically notify me of updates. It seems Intellij can auto perform the alignment for us. In that case, the only downside I can see is if a new contributor attempts to realign it manually after making a change in the table.